package-lock.json

```diff
@@ -6,16 +6,19 @@
     "": {
       "name": "remix-app",
       "dependencies": {
-        "@netlify/remix-edge-adapter": "^3.1.0",
-        "@netlify/remix-runtime": "^2.1.0",
+        "@netlify/edge-functions": "^2.3.1",
+        "@netlify/functions": "^2.6.0",
+        "@netlify/remix-adapter": "^2.3.1",
+        "@netlify/remix-edge-adapter": "^3.2.0",
+        "@netlify/remix-runtime": "^2.2.0",
+        "@remix-run/node": "*",
```

Comment:
It's curious that this diff doesn't match the diff in `package.json` 🤔

Reply:
The changes look to match the expected output, it looks like it was just a little out of sync to start with.

Reply:
@mrstork Ah, never mind — I see why. If I just pull `main` and run `npm install`, the `package-lock.json` changes. It seems it's grown stale and this repo doesn't have any CI checks in place to prevent it.